services/keyboard/src/org/chromium/mojo/keyboard/InputConnectionAdaptor.java

```diff
@@ -71,4 +83,20 @@ public boolean setSelection(int start, int end) {
         mClient.setSelection(start, end);
         return super.setSelection(start, end);
     }
+
+    // Number keys come through as key events instead of commitText!?
```

Comment:
Rather than leave this (and the comment on line 91) as questions, I would recommend phrasing them as descriptions of what you found, including more detail like how you determined this, what specs you looked at to see if it matched expectations or designs, etc.